include/inviwo/core/properties/minmaxproperty.h

```diff
@@ -272,29 +272,27 @@ MinMaxProperty<T>& MinMaxProperty<T>::setMinSeparation(const T& newMinSeparation
 }
 
 template <typename T>
-MinMaxProperty<T>& MinMaxProperty<T>::setRange(const range_type& value) {
-    const auto newRange = clamp({glm::min(value.x, value.y), glm::max(value.x, value.y)});
+MinMaxProperty<T>& MinMaxProperty<T>::setRange(const range_type& newRange) {
+    if (range_.update({glm::min(newRange.x, newRange.y), glm::max(newRange.x, newRange.y)})) {
```

Comment:
Perhaps add a comment why this is necessary or should be done. Is it not an error if the new range is flipped?

Reply:
This is just to be defensive that the new min is less then the new max we have always done this...